tst/testinstall/zmodnz.tst

```diff
@@ -9,8 +9,36 @@
 ##
 gap> START_TEST("zmodnz.tst");
 
+#
+# ZmodnZ constructor
+#
+gap> ZmodnZ(4);
+(Integers mod 4)
+gap> ZmodnZ(-4);
+(Integers mod 4)
+gap> ZmodnZ(1);
+(Integers mod 1)
+gap> ZmodnZ(2);
+GF(2)
+gap> ZmodnZ(0);
+Integers
+gap> ZmodnZ(-1/2);
+Error, <n> must be an integer
+
+#
+# ZmodpZ constructor
+#
+gap> ZmodpZ(2);
+GF(2)
+gap> ZmodpZ(4);
```

Comment:
If I call 
```
gap> IsPrime(-2);
true
gap> ZmodpZ(-2); 
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `IsBound[]' on 2 arguments at /home/makx/ac/gap/lib/methsel2.g:241 called from
<compiled or corrupted statement>  called from
TYPE_FFE( p ) at /home/makx/ac/gap/lib/ffe.gi:219 called from
FFEFamily( p ) at /home/makx/ac/gap/lib/zmodnz.gi:928 called from
ZmodpZNC( p ) at /home/makx/ac/gap/lib/zmodnz.gi:917 called from
<function "ZmodpZ">( <arguments> )
 called from read-eval loop at *stdin*:7
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
```
I think its obvious what happens, do we want to fix it in this PR?